### PR TITLE
ONE 20 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/axiom_engraver.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Axiom Engraver
+ManaCost:1 R
+Types:Creature Phyrexian Wizard
+PT:1/3
+K:etbCounter:OIL:2
+A:AB$ Draw | Cost$ T SubCounter<1/OIL> | SpellDescription$ Draw a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/axiom_engraver.txt:6]: cost incomplete...?

File: forge-gui/res/cardsfolder/upcoming/evolving_adaptive.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Name:Evolving Adaptive
+ManaCost:G
+Types:Creature Phyrexian Warrior
+PT:0/0
+K:etbCounter:OIL:1
+S:Mode$ Continuous | Affected$ Card.Self | AddPower$ X | AddToughness$ X | Description$ CARDNAME gets +1/+1 for each oil counter on it.
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.YouCtrl+powerGTY+Other,Creature.YouCtrl+toughnessGTZ+Other | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever another creature enters the battlefield under your control, if that creature has greater power or toughness than CARDNAME, put an oil counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | CounterType$ OIL
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/evolving_adaptive.txt:8]: intervening if not handled

File: forge-gui/res/cardsfolder/upcoming/forgehammer_centurion.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name:Forgehammer Centurion
+ManaCost:2 R
+Types:Creature Phyrexian Warrior
+PT:3/2
+K:etbCounter:OIL:2
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.YouCtrl+Other,Artifact.YouCtrl+Other | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever another creature or artifact you control is put into a graveyard from the battlefield, put an oil counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ OIL | CounterNum$ 1
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPump | TriggerDescription$ Whenever CARDNAME attacks, you may remove two oil counters from it. When you do, target creature can't block this turn.
+SVar:TrigPump:AB$ Pump | Cost$ SubCounter<2/OIL> | ValidTgts$ Creature | TgtPrompt$ Select target creature | IsCurse$ True | KW$ HIDDEN CARDNAME can't block.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/forgehammer_centurion.txt:9]: reflexive trigger missing

File: forge-gui/res/cardsfolder/upcoming/evolving_adaptive.txt (hunk 2 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/evolving_adaptive.txt:8]: Should this always trigger even if power and toughness is less when it etb's ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/evolving_adaptive.txt:8]: > If the ability triggers, the stat comparison will happen again when the ability tries to resolve. If neither stat of the new creature is greater, the ability will do nothing. If the creature that entered the battlefield leaves the battlefield before the ability tries to resolve, use its last known power and toughness to compare the stats.

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/evolving_adaptive.txt:8]: Ah I see, thanks for the heads up

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/evolving_adaptive.txt]: Like this ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/evolving_adaptive.txt]: No reason to change the trigger event

you can read up on the rule here:
https://yawgatog.com/resources/magic-rules/#R6034

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/evolving_adaptive.txt]: Ah It checks 2 times

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/evolving_adaptive.txt]: getting there, but if you read closely the YouCtrl is actually not part of the condition (belongs only to the trigger event)

**Review by tool4ever — APPROVED** [review]

**Pull request merged**